File: ThreatModelDfdService/Data/DTO/DfdElementResponseDTO.cs
using ThreatModelDfdService.Model.Enums;

namespace ThreatModelDfdService.Data.DTO;

public record DfdElementResponseDTO(
    long Id,
    string Name,
    DfdElementType Type,
    decimal XValue,
    decimal YValue,
    decimal Width,
    decimal Height,
    long? DfdChildId,
    string Uuid
);

File: dfd-service/src/main/java/com/tcc/dfd_service/dto/DfdElementResponseDTO.java
package com.tcc.dfd_service.dto;

import com.tcc.dfd_service.enums.DfdElementType;
import java.math.BigDecimal;

public record DfdElementResponseDTO(
        Long id,
        String name,
        DfdElementType type,
        BigDecimal xValue,
        BigDecimal yValue,
        BigDecimal width,
        BigDecimal height,
        Long dfdChildId,
        String uuid
) {
}